### DISCUSSION TITLE
change energy plan on custom fan and power settings

### DISCUSSION THREAD

**Discussion by spXel** [seerge/g-helper#2759] [Q&A]
Hello, is there a way to put a specific power plan in a especific custom power+fan settings?

**Comment by seerge** [comment]
@spXel yes, there is a way https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode

**Reply by spXel** [reply]
that one i know, but this type of profile wich number is?
![Screenshot_1](https://github.com/seerge/g-helper/assets/102599825/d733776d-bb08-4f91-9c10-2c160a8f3d29)

**Reply by seerge** [reply]
@spXel you can check ID assigned to this mode in the config. Most likely it's the next number, i.e. ``3``

**Reply by spXel** [reply]
I got it! thax a lot!
